src/scratchtocatrobat/converter/converter.py

```diff
@@ -354,6 +355,15 @@ def create_script(cls, scratch_script_name, arguments, catrobat_project, sprite,
             my_script.formulaMap = when_cond_brick.formulaMap
             return my_script
 
+        if scratch_script_name == scratch.SCRIPT_WHEN_BACKGROUND_SWITCHES_TO:
+            background = catrobat.background_sprite_of(catrobat_project.getDefaultScene())
```

Comment:
The implementation works fine, and behaves as expected.
But there is one exceptional case where the conversion of the block fails.
This happens when the WhenSceneStarts (or also named as SCRIPT_WHEN_BACKGROUND_SWITCHES_TO) script is added to the stage/backdrop object.

In order to fix this, please add the following line just right before the assert statement here:
`            background = sprite if background is None else background`

... and don't forget to create a new test as well.